Dockerfile

```diff
@@ -43,11 +43,26 @@ ENV INSTALL_BUNDLED_SNAPSHOTS=true
 
 RUN apt-get -y update && apt-get install -y groovy && rm -rf /var/lib/apt/lists/*
 
+RUN curl -L --show-error https://download.java.net/java/GA/jdk11/13/GPL/openjdk-11.0.1_linux-x64_bin.tar.gz --output openjdk.tar.gz && \
```

Comment:
Would be nice to put 11.0.1 and 13 to ARG definitions so that we can update easily

Reply:
For the moment, I don't know how the next update release will be named and how the URL would be. 

Also, we would have to change the `sha256`  to validate the binary. So when we will change the binary url, we would also have to change the checksum.

Reply:
@oleg-nenashev I can still make the modification if you want. Do you think it's preferable?